apps/web/src/App.tsx

```diff
@@ -255,4 +281,34 @@ function AuthHandlerComponent({ children }: { children: React.ReactNode }) {
   );
 }
 
+function SpotLightProvider({ children }) {
+  const [items, setItems] = useState<SpotlightItem[]>([]);
+
+  const addItem = (item: SpotlightItem | SpotlightItem[]) => {
+    if (!Array.isArray(item)) {
+      item = [item];
+    }
+
+    const newItems = [...items, ...item];
+    newItems.sort((a, b) => {
+      if ((a.order || 10000000) > (b.order || 10000000)) {
```

Comment:
Where can someone add their preferred `order`?

Reply:
when they add the item to the spotlight 😄

Reply:
```suggestion
      newItems.sort((a,b) => (b.order || 10000000) - (a.order || 10000000))
```
Maybe we can shorten the sort comparator to this. wdyt?